javascript/javascript3/week3/PromiseAll/repositories.js

```diff
@@ -0,0 +1,40 @@
+// When you have the data for the different repositories, render the fullname of the repo, url of the repo, and the owner of the repo. See github-repos as an example of how the renderered repos should look. You are more than welcome to style it a bit nicer!
+//github url
+
+const userUrl = "https://api.github.com/search/repositories?q=user:";
+
+const body = document.querySelector("body");
+
+//Fetching data
+const fetchAll = (userName) =>
+    fetch(userUrl + userName)
+    .then((response) => response.json())
+    .then((data) => data.items);
+
+const usersPromises = [
+    fetchAll("anilMula"),
+    fetchAll("merihnguse"),
+    fetchAll("Deepti-Dk"),
+    fetchAll("benna100"),
```

Comment:
Good to see that you have stored all functions in to a variable and then use it at Promise.all 👏